README.md

```diff
@@ -1,21 +1,21 @@
 # Cron expressions parser
 
-The goal of this project is to build a cron expression parser that can parse a cron entry string, print on the console a table and expand each field to show the times at which it will run
+The goal of this project is to build a cron expression parser that can parse a cron entry string, print a table to the console and expand each field to show the times at which it will run.
 
 ## Implementation
 
 I preferred a functional approach to implement the business logic, the parts of which are expressed as pure functions which can be composed in order to return the expected values. 
-Hence, they operate in an Either[A,B] context to represent potential failures. Where it made sense, to provide parallel validation of the cron line tokens, they operate in a ValidatedNec[A,B] context
+Hence, they operate in an Either[A,B] context to represent potential failures. Where it made sense, to provide parallel validation of the cron line tokens, they operate in a ValidatedNec[A,B] context.
 
 ## Build
 
-In a terminal, execute `sbt assembly` from the root of the project, that will generate an executable fat jar, that you'll find in `target/scala-2.13/cron-parser-assembly-<version>.jar`
+In a terminal, execute `sbt assembly` from the root of the project, that will generate an executable fat jar, that you'll find in `target/scala-2.13/cron-parser-assembly-<version>.jar`.
 
 ## Run
 
 Run the application in a terminal using the following command line: `scala target/scala-2.13/cron-parser-assembly-0.1.jar <input cron line>` where <input cron line> is the line that's going to be 
 evaluated.
-Alternatively, you might use the command `java -jar target/scala-2.13/cron-parser-assembly-<version>.jar <input cron line>` if you incur in some issue with running it using the scala executable wrapper
+Alternatively, you might use the command `java -jar target/scala-2.13/cron-parser-assembly-<version>.jar <input cron line>` if you incur issues running it using the scala executable wrapper.
```

Comment:
Shouldn't this "incur in issues" rather than "incur issues"?

Reply:
I'm pretty sure that's how it's used, but I double checked just to make sure: https://sentence.yourdictionary.com/incur

Reply:
Thanks!